README.md

```diff
@@ -31,8 +31,6 @@ Set the environment variables
 
 * Add code for managing rolesmapping
 * Add code for managing actiongroups
-* Allow password customization (e.g. length)
```

Comment:
With this "feature" I actually meant both the option to add custom passwords and to pass arguments to the password generator (like the length)

Reply:
OK, maybe let's add that later.